excellent/functions.go

```diff
@@ -939,10 +941,22 @@ func Field(env utils.Environment, args ...interface{}) interface{} {
 	}
 
 	fields := strings.Split(source, sep)
-	if field-1 >= len(fields) {
+	if field >= len(fields) {
 		return ""
 	}
-	return strings.TrimSpace(fields[field-1])
+
+	// when using spaces as a delimiter, we consider it splitting on whitespace, so remove empty values
+	if len(strings.TrimSpace(sep)) == 0 {
```

Comment:
Should we only be special casing `" "` maybe? Seems slightly weird to be doing this automatically for tabs and multiple spaces.